app/src/main/java/com/pavesid/androidacademy/ui/fragments/MoviesDetailsFragment.kt

```diff
@@ -0,0 +1,115 @@
+package com.pavesid.androidacademy.ui.fragments
+
+import android.graphics.drawable.Drawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import coil.load
+import com.pavesid.androidacademy.R
+import com.pavesid.androidacademy.data.local.FakeRepository
+import com.pavesid.androidacademy.databinding.FragmentMoviesDetailsBinding
+import com.pavesid.androidacademy.ui.activities.MainActivity
+import com.pavesid.androidacademy.ui.adapters.CastAdapter
+import com.pavesid.androidacademy.ui.decorations.CastItemDecoration
+import com.pavesid.androidacademy.utils.Utils
+
+class MoviesDetailsFragment : Fragment() {
+
+    private var _binding: FragmentMoviesDetailsBinding? = null
+    private val binding: FragmentMoviesDetailsBinding
+        get() = _binding!!
+
+    private val castAdapter by lazy { CastAdapter() }
+
+    private var addTitle = ""
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            addTitle = it.getString(PARAM_TITLE, "")
+        }
```

Comment:
i believe you should provide `movie id` and some additional information to this fragment and load all details from Repository